zipkin-api.yaml

```diff
@@ -0,0 +1,259 @@
+swagger: "2.0"
+info:
+  version: "1.0.0"
+  title: Zipkin API
+  description: |
+    Zipkin's Query api is rooted at `api/v1`, on a host that by default listens
+    on port 9411. It primarily serves zipkin-web, although it includes a POST
+    endpoint that can receive spans.
+host: localhost:9411
+basePath: /api/v1
+schemes:
+  - http
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /services:
+    get:
+      description: |
+        Returns a list of all service names associated with annotations.
+      responses:
+        200:
+          description: Succes
+          schema:
+            type: array
+            items:
+              type: string
+        400:
+          description: Bad Request Error
+  /spans:
+    get:
+      parameters:
+      - name: serviceName
+        in: query
+        required: true
+        description: |
+          Ex zipkin-web (required) - service that logged an annotation in the
+          span.
+        type: string
+      responses:
+        200:
+          description: OK
```

Comment:
ps this and /services are lexicographically sorted.. should this be somewhere? ex. here?